src/smooch-node.js

```diff
@@ -64,6 +65,7 @@ export class Smooch extends SmoochBase {
             this.appUsers = new AppUsersApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
             this.conversations = new ConversationsApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
             this.stripe = new StripeApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
+            this.attachments = new AttachmentsApi(this.serviceUrl, this.authHeaders, this.headers, true, this.httpAgent);
```

Comment:
That's a good question...
I believe it's because those APIs are limited to API only (can't be used in the frontend), so that ensures they will only exist on the `smoochCore` object when it's created in Node.js.

This means that `attachments` should also only be in here...
I'll re-test having it only be in this file, last time I tried it didn't work.